lib/gollum-lib/wiki.rb

```diff
@@ -93,8 +93,8 @@ def default_options
     #           :mathjax       - Set to false to disable mathjax.
     #           :user_icons    - Enable user icons on the history page. [gravatar, identicon, none].
     #                            Default: none
-    #           :show_all      - Show all files in file view, not just valid pages.
-    #                            Default: false
+    #           :global_tag_lookup        - Enable 4.x compatibility behavior for links
+    #           :github_tag_compatibility - Enable github compatibility behavior for links (substitute - for spaces in page names)
```

Comment:
I would prefer having a name that does not include github, as gollum is also used by other organizations (notably gitlab). Something like `hyphenate-tags` perhaps or `hyphenated-tag-lookup`?

Reply:
Ah yes, much better! Or `hyphened-tag-lookup` as its slightly simpler?

Reply:
Yep, fine.